hub/src/server/authentication.ts

```diff
@@ -349,19 +354,19 @@ export class LegacyAuthentication {
     return Buffer.from(JSON.stringify(authObj)).toString('base64')
   }
 
-  getAuthenticationScopes() : Array<AuthScopeType> {
+  getAuthenticationScopes(): Array<AuthScopeType> {
     // no scopes supported in this version
     return []
   }
 
   isAuthenticationValid(address: string, challengeTexts: Array<string>,
-                        options? : {}) { //  eslint-disable-line no-unused-vars
+                        options? : {}) { //  eslint-disable-line @typescript-eslint/no-unused-vars
     if (ecPairToAddress(this.publickey) !== address) {
       throw new ValidationError('Address not allowed to write on this path')
     }
 
     for (const challengeText of challengeTexts) {
-      const digest = bitcoin.crypto.sha256(challengeText)
+      const digest = bitcoin.crypto.sha256(Buffer.from(challengeText))
```

Comment:
Interesting.  Does this work the same as before, without `Buffer.from`?

Reply:
The type definitions only accept a Buffer: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bitcoinjs-lib/index.d.ts#L342

Looking through the source it looks like it ends up using node's hash.update API https://nodejs.org/api/crypto.html#crypto_hash_update_data_inputencoding which does accept a string.

Reply:
Yeah, this works the same as before (the auth unit tests would be failing if it changed).